podaac/subsetter/xarray_enhancements.py

```diff
@@ -68,10 +68,16 @@ def get_indexers_from_nd(cond: xr.Dataset, cut: bool) -> dict:
     dict
         Indexer dictionary for the provided condition.
     """
-
-    rows = np.any(cond.values.squeeze(), axis=1)
+    # check if the lat/lon coordinate numpy array has 2 or more dimensions
+    if cond.values.squeeze().ndim == 2:
+        x_axis = 1
+        y_axis = 0
+    else:
```

Comment:
is this intended to be for both one-dimensional _and_ more than two dimensions?

Reply:
@danielfromearth 1d array is already handled in line 30 function. This would be for more than 2 dimensions